pkgs/by-name/pc/pcsx2/shaderc-patched.nix

```diff
@@ -1,20 +1,35 @@
 {
+  lib,
+  fetchFromGitHub,
   fetchpatch,
   pcsx2,
   shaderc,
 }:
 
+let
+  version = "2024.1";
+in
 shaderc.overrideAttrs (old: {
+  inherit version;
   pname = "shaderc-patched-for-pcsx2";
+  src = fetchFromGitHub {
+    owner = "google";
+    repo = "shaderc";
+    rev = "v${version}";
```

Comment:
```suggestion
    rev = "refs/tags/v${version}";
```

Reply:
Is this causing bugs?

Reply:
No, but `"v${version}"` could refer to a branch if one were created with that name.

Reply:
Upstream could also just force‐push a different tag pointer if they wanted to break things, right? Not objecting to the principle, just wondering if this has ever come up in practice.

Reply:
My concern is more about the ambiguity than the possibility of breaking stuff. If they wanted to guarantee a breakage, they could delete the repo.

Reply:
I remember this happening a long time ago with nickel.
But I think we should not be overzealous with this.